main.go

```diff
@@ -65,6 +65,7 @@ func main() {
 				SecurityAlerts:        c.Bool("security-alerts"),
 				EventsFile:            os.Stdout,
 				ErrorsFile:            os.Stderr,
+				StackTraces:           c.Bool("stack-traces"),
```

Comment:
Please see my other comment.
What I actually meant is adding this option to the capture flag